compiler/optimizer/VPHandlers.cpp

```diff
@@ -1419,15 +1419,15 @@ static const char *getFieldSignature(OMR::ValuePropagation *vp, TR::Node *node,
    return NULL;
    }
 
-static void addKnownObjectConstraints(OMR::ValuePropagation *vp, TR::Node *node)
```

Comment:
Maybe a comment here to indicate what true and false mean to consumers would be good?